viomi.v7 (Mop P) logs spammed with  unhandledRejection MiioTimeoutError: request timed out:{"method":"set_timezone",

<!-- Text between these tags doesn't get displayed so you don't have to remove it -->
Yesterday's pr installed, thousands of errors [ERROR] unhandledRejection MiioTimeoutError: request timed out:{"method":"set_timezone","params":[0],"id":xxxx}
<!--
A clear and concise description of what the bug is.
-->

## To Reproduce
<!--
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
-->
Robot has valetudo installed and MQTT active. No internet access is given to the robot network.
This error appears every 10 seconds in the logs.

### Screenshots
<!--If applicable, add screenshots to help explain your problem.-->
n.a.
### Vacuum Model
<!--  -->
viomi v7 (Xiaomi Mi Robot Vacuum-Mop P)

### Valetudo Version
<!-- -->
2021.03.00 + pr 25836e...

## Expected behavior
<!--
A clear and concise description of what you expected to happen.
-->
No errors :)

## Additional context
<!--  -->
partial log (log level: trace) attached for a single msg id:2767

> [2021-03-12T20:40:56.445Z] [DEBUG] >>> cloud: {"method":"set_timezone","params":[0],"id":2767}
> [2021-03-12T20:40:57.088Z] [TRACE] <<< cloud: {"result":["ok"],"id":2765}
> [2021-03-12T20:40:57.096Z] [DEBUG] map_upload_handler { '2755629254858': '', suffix: 'urls' }
> [2021-03-12T20:40:57.115Z] [TRACE] fpos:0 take 2@featureFlags <Buffer ff 78>
> [2021-03-12T20:40:57.116Z] [TRACE] fpos:2 take 2@unknown01 <Buffer 00 00>
> [2021-03-12T20:40:57.117Z] [TRACE] fpos:4 take 4@mapId <Buffer 1c d4 4d 60>
> [2021-03-12T20:40:57.117Z] [TRACE] fpos:8 take 40@robot status <Buffer 01 00 00 00 00 00 00 00 c8 00 00 00 01 00 00 00 03 00 00 00 39 08 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:57.118Z] [TRACE] fpos:30 take 40@map head <Buffer 1c d4 4d 60 01 00 00 00 01 00 00 00 20 03 00 00 20 03 00 00 00 00 a0 c1 00 00 a0 c1 00 00 a0 41 00 00 a0 41 cd cc 4c 3d>
> [2021-03-12T20:40:57.154Z] [TRACE] fpos:9c458 take 12@history <Buffer 1c d4 4d 60 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:57.155Z] [TRACE] fpos:9c464 take 16@charge station <Buffer 1c d4 4d 60 a0 60 2e be e4 d2 e3 3d 0a e8 38 bf>
> [2021-03-12T20:40:57.156Z] [TRACE] fpos:9c474 take 12@virtual wall <Buffer 1c d4 4d 60 00 00 00 00 04 00 00 00>
> [2021-03-12T20:40:57.157Z] [TRACE] fpos:9c480 take 12@virtual wall prefix <Buffer 01 00 00 00 00 00 00 00 04 00 00 00>
> [2021-03-12T20:40:57.157Z] [TRACE] fpos:9c48c take 32@Virtual walls coords <Buffer 9a 99 89 40 33 33 63 c0 9a 99 89 40 33 33 63 c0 00 00 a8 40 33 33 63 c0 00 00 a8 40 33 33 63 c0>
> [2021-03-12T20:40:57.158Z] [TRACE] fpos:9c4ac take 48@unknown48 <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:57.158Z] [TRACE] fpos:9c4dc take 12@virtual wall prefix <Buffer 02 00 00 00 00 00 00 00 04 00 00 00>
> [2021-03-12T20:40:57.159Z] [TRACE] fpos:9c4e8 take 32@Virtual walls coords <Buffer 00 00 00 40 cd cc 7c c0 33 33 73 40 cd cc 7c c0 33 33 73 40 cd cc 8c c0 00 00 00 40 cd cc 8c c0>
> [2021-03-12T20:40:57.159Z] [TRACE] fpos:9c508 take 48@unknown48 <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:57.160Z] [TRACE] fpos:9c538 take 12@virtual wall prefix <Buffer 03 00 00 00 00 00 00 00 04 00 00 00>
> [2021-03-12T20:40:57.161Z] [TRACE] fpos:9c544 take 32@Virtual walls coords <Buffer 66 66 e6 3e 00 00 40 3f 66 66 66 40 00 00 40 3f 66 66 66 40 33 33 b3 3e 66 66 e6 3e 33 33 b3 3e>
> [2021-03-12T20:40:57.161Z] [TRACE] fpos:9c564 take 48@unknown48 <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:57.162Z] [TRACE] fpos:9c594 take 12@virtual wall prefix <Buffer 04 00 00 00 00 00 00 00 04 00 00 00>
> [2021-03-12T20:40:57.162Z] [TRACE] fpos:9c5a0 take 32@Virtual walls coords <Buffer 9a 99 19 be 9a 99 b9 bf 9a 99 59 3f 9a 99 b9 bf 9a 99 59 3f 00 00 20 c0 9a 99 19 be 00 00 20 c0>
> [2021-03-12T20:40:57.163Z] [TRACE] fpos:9c5c0 take 48@unknown48 <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:57.163Z] [TRACE] fpos:9c5f0 take 12@area head <Buffer 1c d4 4d 60 01 00 00 00 00 00 00 00>
> [2021-03-12T20:40:57.164Z] [TRACE] fpos:9c5fc take 20@navigate <Buffer 1c d4 4d 60 00 00 00 00 00 80 89 44 00 80 89 44 00 00 00 00>
> [2021-03-12T20:40:57.167Z] [TRACE] fpos:9c610 take 21@realtime <Buffer 1c d4 4d 60 00 00 00 00 00 ae cf 90 be 27 75 57 3e 0a e8 38 bf>
> [2021-03-12T20:40:57.167Z] [TRACE] fpos:9c625 take 8@unknown8 <Buffer 1c d4 4d 60 00 00 00 01>
> [2021-03-12T20:40:57.168Z] [TRACE] fpos:9c62d take 4@room header <Buffer 1c d4 4d 60>
> [2021-03-12T20:40:57.169Z] [TRACE] fpos:9c636 take 4@mapArg:Map1 <Buffer 01 00 00 00>
> [2021-03-12T20:40:57.169Z] [TRACE] fpos:9c63a take 4@room length <Buffer 05 00 00 00>
> [2021-03-12T20:40:57.170Z] [TRACE] fpos:9c63e room10: room0 640574
> [2021-03-12T20:40:57.170Z] [TRACE] fpos:9c64e room11: room3 640590
> [2021-03-12T20:40:57.170Z] [TRACE] fpos:9c65e room12: room1 640606
> [2021-03-12T20:40:57.171Z] [TRACE] fpos:9c66e room13: room2 640622
> [2021-03-12T20:40:57.171Z] [TRACE] fpos:9c67e room14: room4 640638
> [2021-03-12T20:40:57.171Z] [TRACE] fpos:9c68e take 6@unknown <Buffer 01 01 00 00 00 00>
> [2021-03-12T20:40:57.172Z] [TRACE] fpos:9c694 take 12@rooms2head <Buffer 1c d4 4d 60 01 00 00 00 04 00 00 00>
> [2021-03-12T20:40:57.172Z] [TRACE] fpos:9c6a0 take 4@rooms2 length <Buffer 01 00 00 00>
> [2021-03-12T20:40:57.173Z] [TRACE] fpos:9c6a4 take 2@room data <Buffer 00 00>
> [2021-03-12T20:40:57.173Z] [TRACE] fpos:9c6a6 take 4@unknown2 <Buffer 00 00 04 00>
> [2021-03-12T20:40:57.174Z] [TRACE] fpos:9c6a7 take 50@unknown50 <Buffer 00 04 00 00 00 9a 99 89 40 33 33 63 c0 9a 99 89 40 33 33 63 c0 00 00 a8 40 33 33 63 c0 00 00 a8 40 33 33 63 c0 00 00 00 00 00 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:57.174Z] [TRACE] fpos:9c6d9 take 5@unknown5 <Buffer 00 00 00 00 00>
> [2021-03-12T20:40:57.175Z] [TRACE] fpos:9c6de take 4@pose length <Buffer 00 00 00 00>
> [2021-03-12T20:40:57.175Z] [TRACE] fpos:9c6e2 take 5460@trailing <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 04 00 00 00 00 00 00 40 cd cc 7c c0 33 33 73 40 ... 5410 more bytes>
> [2021-03-12T20:40:57.196Z] [DEBUG] <<< cloud: {"result":["ok"],"id":2765}
> [2021-03-12T20:40:57.197Z] [INFO] << cloud: ignoring response for non-pending request {"result":["ok"],"id":2765}
> [2021-03-12T20:40:57.523Z] [TRACE] >>> cloud: {"method":"set_uploadmap","params":[2],"id":2768}
> [2021-03-12T20:40:58.207Z] [TRACE] <<< cloud: {"result":["ok"],"id":2768}
> [2021-03-12T20:40:58.306Z] [DEBUG] map_upload_handler { '2755629254858': '', suffix: 'urls' }
> [2021-03-12T20:40:58.326Z] [TRACE] fpos:0 take 2@featureFlags <Buffer ff 78>
> [2021-03-12T20:40:58.327Z] [TRACE] fpos:2 take 2@unknown01 <Buffer 00 00>
> [2021-03-12T20:40:58.328Z] [TRACE] fpos:4 take 4@mapId <Buffer 1c d4 4d 60>
> [2021-03-12T20:40:58.329Z] [TRACE] fpos:8 take 40@robot status <Buffer 01 00 00 00 00 00 00 00 c8 00 00 00 01 00 00 00 03 00 00 00 39 08 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:58.329Z] [TRACE] fpos:30 take 40@map head <Buffer 1c d4 4d 60 01 00 00 00 01 00 00 00 20 03 00 00 20 03 00 00 00 00 a0 c1 00 00 a0 c1 00 00 a0 41 00 00 a0 41 cd cc 4c 3d>
> [2021-03-12T20:40:58.367Z] [TRACE] fpos:9c458 take 12@history <Buffer 1c d4 4d 60 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:58.368Z] [TRACE] fpos:9c464 take 16@charge station <Buffer 1c d4 4d 60 a0 60 2e be e4 d2 e3 3d 0a e8 38 bf>
> [2021-03-12T20:40:58.369Z] [TRACE] fpos:9c474 take 12@virtual wall <Buffer 1c d4 4d 60 00 00 00 00 04 00 00 00>
> [2021-03-12T20:40:58.369Z] [TRACE] fpos:9c480 take 12@virtual wall prefix <Buffer 01 00 00 00 00 00 00 00 04 00 00 00>
> [2021-03-12T20:40:58.370Z] [TRACE] fpos:9c48c take 32@Virtual walls coords <Buffer 9a 99 89 40 33 33 63 c0 9a 99 89 40 33 33 63 c0 00 00 a8 40 33 33 63 c0 00 00 a8 40 33 33 63 c0>
> [2021-03-12T20:40:58.370Z] [TRACE] fpos:9c4ac take 48@unknown48 <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:58.371Z] [TRACE] fpos:9c4dc take 12@virtual wall prefix <Buffer 02 00 00 00 00 00 00 00 04 00 00 00>
> [2021-03-12T20:40:58.372Z] [TRACE] fpos:9c4e8 take 32@Virtual walls coords <Buffer 00 00 00 40 cd cc 7c c0 33 33 73 40 cd cc 7c c0 33 33 73 40 cd cc 8c c0 00 00 00 40 cd cc 8c c0>
> [2021-03-12T20:40:58.372Z] [TRACE] fpos:9c508 take 48@unknown48 <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:58.373Z] [TRACE] fpos:9c538 take 12@virtual wall prefix <Buffer 03 00 00 00 00 00 00 00 04 00 00 00>
> [2021-03-12T20:40:58.374Z] [TRACE] fpos:9c544 take 32@Virtual walls coords <Buffer 66 66 e6 3e 00 00 40 3f 66 66 66 40 00 00 40 3f 66 66 66 40 33 33 b3 3e 66 66 e6 3e 33 33 b3 3e>
> [2021-03-12T20:40:58.374Z] [TRACE] fpos:9c564 take 48@unknown48 <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:58.375Z] [TRACE] fpos:9c594 take 12@virtual wall prefix <Buffer 04 00 00 00 00 00 00 00 04 00 00 00>
> [2021-03-12T20:40:58.375Z] [TRACE] fpos:9c5a0 take 32@Virtual walls coords <Buffer 9a 99 19 be 9a 99 b9 bf 9a 99 59 3f 9a 99 b9 bf 9a 99 59 3f 00 00 20 c0 9a 99 19 be 00 00 20 c0>
> [2021-03-12T20:40:58.376Z] [TRACE] fpos:9c5c0 take 48@unknown48 <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:58.377Z] [TRACE] fpos:9c5f0 take 12@area head <Buffer 1c d4 4d 60 01 00 00 00 00 00 00 00>
> [2021-03-12T20:40:58.377Z] [TRACE] fpos:9c5fc take 20@navigate <Buffer 1c d4 4d 60 00 00 00 00 00 80 89 44 00 80 89 44 00 00 00 00>
> [2021-03-12T20:40:58.378Z] [TRACE] fpos:9c610 take 21@realtime <Buffer 1c d4 4d 60 00 00 00 00 00 ae cf 90 be 27 75 57 3e 0a e8 38 bf>
> [2021-03-12T20:40:58.378Z] [TRACE] fpos:9c625 take 8@unknown8 <Buffer 1c d4 4d 60 00 00 00 01>
> [2021-03-12T20:40:58.379Z] [TRACE] fpos:9c62d take 4@room header <Buffer 1c d4 4d 60>
> [2021-03-12T20:40:58.379Z] [TRACE] fpos:9c636 take 4@mapArg:Map1 <Buffer 01 00 00 00>
> [2021-03-12T20:40:58.380Z] [TRACE] fpos:9c63a take 4@room length <Buffer 05 00 00 00>
> [2021-03-12T20:40:58.380Z] [TRACE] fpos:9c63e room10: room0 640574
> [2021-03-12T20:40:58.381Z] [TRACE] fpos:9c64e room11: room3 640590
> [2021-03-12T20:40:58.381Z] [TRACE] fpos:9c65e room12: room1 640606
> [2021-03-12T20:40:58.381Z] [TRACE] fpos:9c66e room13: room2 640622
> [2021-03-12T20:40:58.381Z] [TRACE] fpos:9c67e room14: room4 640638
> [2021-03-12T20:40:58.381Z] [TRACE] fpos:9c68e take 6@unknown <Buffer 01 01 00 00 00 00>
> [2021-03-12T20:40:58.382Z] [TRACE] fpos:9c694 take 12@rooms2head <Buffer 1c d4 4d 60 01 00 00 00 04 00 00 00>
> [2021-03-12T20:40:58.382Z] [TRACE] fpos:9c6a0 take 4@rooms2 length <Buffer 01 00 00 00>
> [2021-03-12T20:40:58.383Z] [TRACE] fpos:9c6a4 take 2@room data <Buffer 00 00>
> [2021-03-12T20:40:58.383Z] [TRACE] fpos:9c6a6 take 4@unknown2 <Buffer 00 00 04 00>
> [2021-03-12T20:40:58.384Z] [TRACE] fpos:9c6a7 take 50@unknown50 <Buffer 00 04 00 00 00 9a 99 89 40 33 33 63 c0 9a 99 89 40 33 33 63 c0 00 00 a8 40 33 33 63 c0 00 00 a8 40 33 33 63 c0 00 00 00 00 00 00 00 00 00 00 00 00 00>
> [2021-03-12T20:40:58.384Z] [TRACE] fpos:9c6d9 take 5@unknown5 <Buffer 00 00 00 00 00>
> [2021-03-12T20:40:58.385Z] [TRACE] fpos:9c6de take 4@pose length <Buffer 00 00 00 00>
> [2021-03-12T20:40:58.385Z] [TRACE] fpos:9c6e2 take 5460@trailing <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 04 00 00 00 00 00 00 40 cd cc 7c c0 33 33 73 40 ... 5410 more bytes>
> [2021-03-12T20:40:58.405Z] [DEBUG] <<< cloud: {"result":["ok"],"id":2768}
> [2021-03-12T20:40:58.406Z] [INFO] << cloud: ignoring response for non-pending request {"result":["ok"],"id":2768}
> [2021-03-12T20:40:58.443Z] [DEBUG] cloud request 2767 set_timezone timed out
> [2021-03-12T20:40:58.446Z] [INFO] Cloud message timed out. Assuming that we're not connected anymore
> [2021-03-12T20:40:58.447Z] [ERROR] unhandledRejection MiioTimeoutError: request timed out:{"method":"set_timezone","params":[0],"id":2767}
>     at Timeout.timeout [as _onTimeout] (/snapshot/Valetudo-devel/lib/miio/MiioSocket.js:197:32)
>     at listOnTimeout (internal/timers.js:554:17)
>     at processTimers (internal/timers.js:497:7)

Definetively robotManager is the responsible for setting a wrong date: after stopping it the date is ok.

I have also seen in other issues that without robotManager the vacuum will not work, so no point there.

In the other hand, the error in the logs is still there: 
[2021-04-02T09:56:22.249Z] [ERROR] unhandledRejection MiioTimeoutError: request timed out:{"method":"set_timezone","params":[0],"id":337}
    at Timeout.timeout [as _onTimeout] (/snapshot/Valetudo-devel/lib/miio/MiioSocket.js:197:32)
    at listOnTimeout (internal/timers.js:554:17)
    at processTimers (internal/timers.js:497:7)

[2021-04-03T10:26:02.937Z] [ERROR] unhandledRejection MiioTimeoutError: request timed out:{"method":"set_timezone","params":[0],"id":29844}
    at Timeout.timeout [as _onTimeout] (/snapshot/Valetudo/lib/miio/MiioSocket.js:197:32)
    at listOnTimeout (internal/timers.js:554:17)
    at processTimers (internal/timers.js:497:7)
since 2021 versions

> `[2021-04-03T10:26:02.937Z] [ERROR] unhandledRejection MiioTimeoutError: request timed out:{"method":"set_timezone","params":[0],"id":29844} at Timeout.timeout [as _onTimeout] (/snapshot/Valetudo/lib/miio/MiioSocket.js:197:32) at listOnTimeout (internal/timers.js:554:17) at processTimers **(internal/timers.js:497:7)`**
> 
> Every 16 seconds in log since #732 , if I am not mistaken
> ViomiV8

Indeed, every 16 seconds and sometimes every 8 seconds (duplicate number of calls?)

Will study #732 and see what can be done to avoid this error.

--- a/lib/robots/viomi/ViomiValetudoRobot.js
+++ b/lib/robots/viomi/ViomiValetudoRobot.js
@@ -149,12 +149,12 @@ class ViomiValetudoRobot extends MiioValetudoRobot {
         super.onCloudConnected();
 
         setTimeout(() => {
-            this.sendCommand("get_prop", ["timezone"]).then((res) => {
+            this.sendCommand("get_prop", ["timezone"], {timeout: 12000}).then((res) => {
                 if (res.length > 0) {
                     const timezone = res[0];
                     if (timezone !== 0) {
                         // Set timezone to UTC
-                        this.sendCommand("set_timezone", [0]).then(_ => {
+                        this.sendCommand("set_timezone", [0], {timeout: 12000}).then(_ => {
                             Logger.info("Viomi timezone adjusted to UTC");
                         });
                     }

